api/src/organizational-entities/infrastructure/repositories/organization-tag.repository.js

```diff
@@ -52,4 +52,8 @@ const getRecentlyUsedTags = async function ({ tagId, numberOfRecentTags }) {
   return tags.map(({ tagId: id, name }) => new Tag({ id, name }));
 };
 
-export { batchCreate, create, getRecentlyUsedTags, isExistingByOrganizationIdAndTagId };
+const deleteTagsByOrganizationId = async function (organizationId) {
+  await knex('organization-tags').where({ organizationId }).delete();
+};
```

Comment:
Créer cette nouvelle fonction en l'équipant de base pour pouvoir être transactionnelle :

```suggestion
const deleteTagsByOrganizationId = async function (organizationId) {
  const knexConn = DomainTransaction.getConnection();
  await knexConn('organization-tags').where({ organizationId }).delete();
};
```